solutions/USACO dec19-gymnastics.mdx

```diff
@@ -68,4 +68,56 @@ public class gymnastics {
 ```
 </JavaSection>
 
-</LanguageSection>
\ No newline at end of file
+<CPPSection>
```

Comment:
move this below `**Time complexity:** $\mathcal{O}(K \cdot N^2)$.`(i think this is breaking the build)